roles/pulp/handlers/main.yml

```diff
@@ -22,3 +22,4 @@
   notify: Restart pulpcore-api.service
   environment:
     PULP_SETTINGS: "{{ pulp_settings_file }}"
+    PULP_STATIC_ROOT: "{{ pulp_webserver_static_dir }}"
```

Comment:
Also, I think we are supposed to update the manual install docs every time we make a change like this to the installer.

Reply:
what do you mean by manual install docs?

Reply:
https://docs.pulpproject.org/

https://github.com/pulp/pulpcore/tree/master/docs

I think we've overlooked making multiple updates to it recently.

Reply:
basically the doc just points to pulp_installer README:
https://docs.pulpproject.org/installation/instructions.html#ansible-installation-recommended

Reply:
That's at the top of it because it is recommended.

But after that, you have a lot more install & config instructions:
https://docs.pulpproject.org/installation/instructions.html#pypi-installation
https://docs.pulpproject.org/installation/configuration.html
https://docs.pulpproject.org/installation/storage.html
https://docs.pulpproject.org/installation/authentication.html

Reply:
@fabricio-aguiar I've been thinking: I suggest we just discuss the lack of multiple manual docs updates at our next CI meeting.

Reply:
+1, but I don't think it should be on CI meeting, I think we need an installer meeting

Reply:
I agree and actually we probably need a 30 min check-in meeting weekly. (sorry to suggest even more meetings)